### ISSUE TITLE
Not working for me since 0.25

### ISSUE THREAD

**Issue by philfontaine** [vscode-contrib/vscode-versionlens#182]
Analyzing is stuck in a never ending loop :

![image](https://user-images.githubusercontent.com/16233697/78915318-bbd1fa80-7a59-11ea-8318-e65ba5f471f4.png)

Working fine if I downgrade to 0.24

This is my package.json

```json
{
  "name": "test",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "dev": "quasar dev",
    "lint": "eslint src --ext .js,.ts,.vue",
    "build": "yarn lint --max-warnings 0 && quasar build"
  },
  "dependencies": {
    "@quasar/extras": "1.6.1",
    "@vue/composition-api": "0.5.0",
    "lodash.keyby": "4.6.0",
    "lodash.set": "^4.3.2",
    "quasar": "1.9.0",
    "vue-i18n": "8.16.0"
  },
  "devDependencies": {
    "@quasar/app": "1.6.1",
    "@types/node": "12.12.29",
    "@typescript-eslint/eslint-plugin": "2.26.0",
    "@typescript-eslint/parser": "2.26.0",
    "@vue/eslint-config-prettier": "6.0.0",
    "babel-eslint": "10.1.0",
    "eslint": "6.8.0",
    "eslint-config-prettier": "6.10.1",
    "eslint-plugin-prettier": "3.1.2",
    "eslint-plugin-vue": "6.2.2",
    "prettier": "2.0.4",
    "xml2js": "0.4.23"
  }
}

```

**Comment by philfontaine** [comment]
Never mind, not working either on 0.24. This seems to be linked to problems mentionned in other issues such as #131.

**Comment by pflannery** [comment]
This works for me in v0.25.0 and is quick to resolve, not sure what the problem is.
![image](https://user-images.githubusercontent.com/1727302/80922834-80d09780-8d77-11ea-852e-d0aa8f39b70d.png)

@philfontaine 
What OS do u use?

Can you open the dev tools in vscode and take a look for any errors?

**Comment by philfontaine** [comment]
@pflannery I'm on Windows 10

I currently cannot reproduce the problem on 0.25 nor on 0.26.

When I did encounter the problem, I opened the dev tools and there were no errors.

**Comment by pflannery** [comment]
0.26.2 has addressed an npm view in #190 issue which led me to find a cache issue in npm. I;ve addressed the issue on this side by rolling back npm and added better error handling. Will close this. thanks for your help
